src/tabs/pid_tuning.html

```diff
@@ -867,6 +703,175 @@
+                                </td>
+                            </tr>
+
+                            <tr class="vbatSagCompensation">
+                                <td><input type="checkbox" id="vbatSagCompensation" class="toggle" /></td>
+                                <td colspan="2">
+                                    <span i18n="pidTuningVbatSagCompensation"></span>
+                                    <div class="helpicon cf_tip" i18n_title="pidTuningVbatSagCompensationHelp"></div>
+
+                                    <span class="vbatSagValue suboption">
+                                        <input type="number" name="vbatSagValue" step="1" min="1" max="150" />
+                                        <label for="vbatSagValue">
+                                            <span i18n="pidTuningVbatSagValue"></span>
+                                        </label>
+                                    </span>
+                                </td>
+                            </tr>
+
+                            <tr class="thrustLinearization">
+                                <td><input type="checkbox" id="thrustLinearization" class="toggle" /></td>
+                                <td colspan="2">
+                                    <span i18n="pidTuningThrustLinearization"></span>
+                                    <div class="helpicon cf_tip" i18n_title="pidTuningThrustLinearizationHelp"></div>
+
+                                    <span class="thrustLinearValue suboption">
+                                        <input type="number" name="thrustLinearValue" step="1" min="1" max="150" />
+                                        <label for="thrustLinearValue">
+                                            <span i18n="pidTuningThrustLinearValue"></span>
+                                        </label>
+                                    </span>
+                                </td>
+                            </tr>
+
+                            <tr class="vbatpidcompensation">
+                                <td><input type="checkbox" id="vbatpidcompensation" class="toggle" /></td>
+                                <td colspan="2">
+                                    <span i18n="pidTuningVbatPidCompensation"></span>
+                                    <div class="helpicon cf_tip" i18n_title="pidTuningVbatPidCompensationHelp"></div>
+                                </td>
+                            </tr>
+                        </table>
+                    </div>
+
+                    <div class="gui_box grey pidControllerAdvancedSettings spacer_left">
+                        <table class="pid_titlebar new_rates">
+                            <tr>
+                                <th i18n="pidTuningMiscSettings"></th>
+                            </tr>
+                        </table>
+                        <table class="compensation">
+
+                            <tr class="cellCount">
```

Comment:
Cell count is an adition for motor output limit, so they should go always together.

Reply:
I think they are independent.  We will get confirmation of this hopefully from other experts.

Reply:
We have confirmed that they are independent.